gradle-plugins/src/integTest/groovy/com/linkedin/pegasus/gradle/publishing/PegasusPluginIvyPublishIntegrationTest.groovy

```diff
@@ -0,0 +1,340 @@
+package com.linkedin.pegasus.gradle.publishing
```

Comment:
All new source files should have a license at the top (just copy from any other Java file and update the date).

Reply:
Even for tests? I'm fine with that, but we haven't been following that well based on existing classes in the `test` and `integTest` sourceSets.

Reply:
We add licenses for all Java tests, yes. If the license wasn't added to the other Gradle integ tests, I'd say this was a mistake. I agree that the necessity of this could be re-evaluated, but for now keeping consistency is preferable.